test/integration/_blockchain.js

```diff
@@ -24,7 +24,7 @@ testnet.faucet = function faucet (address, amount, done) {
         if (err) return callback(err)
 
         var unspent = result.filter(function (unspent) {
-          return unspent.value > 1e3
+          return unspent.value > amount
```

Comment:
This isn't right.  You might want `>=` if you're looking to filter non-amount values, which is probably the better way to do it.
I was previously just filtering all dust-like values `<= 1e3`.